content/zh-tw/how-it-works.md

```diff
@@ -26,7 +26,11 @@ Let's Encrypt通過公鑰識別伺服器管理員。 憑證管理軟體（Agent
      src="/images/howitworks_challenge.png"/>
 </div>
 
+<<<<<<< HEAD
+憑證管理軟體（Agent）需要完成所提供的一組挑戰。假設它能夠完成上面的第二個任務：它在`https：// example.com`站點上的指定路徑上創建一個文件。憑證管理軟體（Agent）還使用其私鑰對提供的nonce（特殊密鑰）進行簽名。完成這些步驟後，憑證管理軟體（Agent）會通知CA它已準備好完成驗證。
+=======
 憑證管理軟體（Agent）需要完成所提供的一組挑戰。假設它能夠完成上面的第二個任務：它在`http：// example.com`站點上的指定路徑上創建一個文件。憑證管理軟體（Agent）還使用其私鑰對提供的nonce（特殊密鑰）進行簽名。完成這些步驟後，憑證管理軟體（Agent）會通知CA它已準備好完成驗證。
+>>>>>>> upstream/master
```

Comment:
A small mistake to fix here, the other parts looks good to me!

Reply:
Oh my bad, fixed it right away!